From 94a0a210a485ce1fc95c47a6bc67c4c16c89a0f9 Mon Sep 17 00:00:00 2001
From: Dieter Hametner
Date: Sun, 25 Mar 2012 14:42:12 +0000
Subject: Updated LIVE to latest changes in vdr 1.7.27

- Do not use the direct access to cRecoding members any more.
---
diff --git a/epg_events.cpp b/epg_events.cpp
index a514769..cd4996e 100644
--- a/epg_events.cpp
+++ b/epg_events.cpp
@@ -182,12 +182,20 @@ namespace vdrlive
 
 	time_t EpgRecording::GetStartTime() const
 	{
+#if VDRVERSNUM < 10726
 		return m_recording ? m_recording->start : 0;
+#else
+		return m_recording ? m_recording->Start() : 0;
+#endif
 	}
 
 	time_t EpgRecording::GetEndTime() const
 	{
+#if VDRVERSNUM < 10726
 		return m_recording ? m_recording->start : 0;
+#else
+		return m_recording ? m_recording->Start() : 0;
+#endif
 	}
 
 	const string EpgRecording::Name() const
diff --git a/recman.cpp b/recman.cpp
index fb315d2..6a2fc00 100644
--- a/recman.cpp
+++ b/recman.cpp
@@ -383,7 +383,11 @@ namespace vdrlive {
 
 	time_t RecordingsItemRec::StartTime() const
 	{
+#if VDRVERSNUM < 10726
 		return m_recording->start;
+#else
+		return m_recording->Start();
+#endif
 	}
 
 	long RecordingsItemRec::Duration() const
--
cgit v0.9.0.2-40-g7c79
